cloud-function/src/env.ts

```diff
@@ -71,8 +71,7 @@ export const SIGN_SECRET = process.env["SIGN_SECRET"] ?? "";
 export const BSA_ZONE_KEYS = Object.fromEntries(
   (process.env["BSA_ZONE_KEYS"] ?? "").split(";").map((k) => k.split(":"))
 );
-export const BSA_URL_PREFIX =
-  process.env["BSA_URL_PREFIX"] ?? "https://localhost";
+export const BSA_URL_PREFIX = process.env["BSA_URL_PREFIX"] ?? "";
```

Comment:
```suggestion
```

Reply:
Weird, I swear it didn't pop up in my VSCode search. 🤷

Reply:
Fixed via [ff55991](https://github.com/mdn/yari/pull/10011/commits/ff5599137fc33e9ccc140d2cac54c71a557ebc24).